Lib/test/test_memoryview.py

```diff
@@ -545,6 +545,104 @@ def test_pickle(self):
             with self.assertRaises(TypeError):
                 pickle.dumps(m, proto)
 
+    def test_use_released_memory(self):
+        size = 128
+        def release():
+            m.release()
+            nonlocal ba
+            ba = bytearray(size)
```

Comment:
That's useless, no?

Reply:
No, we need it for tests below that tests indexing into `ba[]`.

Reply:
We allocate a bytearray of the same size as the bytearray just released in memoryview in hope that it will be allocated at the same memory. It helps to check that we do nor read/write a freed memory.